src/benchmarks/micro/libraries/System.Collections/Perf.ObservableCollection.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.ObjectModel;
+
+namespace System.Collections.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.Collections)]
+    public class Perf_ObservableCollection
+    {
+        private readonly ObservableCollection<int> _collection = new ObservableCollection<int>();
```

Comment:
have you considered adding it to the existing benchmarks where we test other collections using similar pattern? https://github.com/dotnet/performance/blob/main/src/benchmarks/micro/libraries/System.Collections/CreateAddAndClear.cs

We would get:
* coverage for `T : class`
* possibility to compare `ObservableCollection` against other collections

Reply:
Done